src/test/java/uk/gov/hmcts/reform/sandl/snlapi/services/EventsCommunicationServiceTest.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.reform.sandl.snlapi.services;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.util.ReflectionTestUtils;
+import org.springframework.web.client.RestTemplate;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(SpringRunner.class)
+public class EventsCommunicationServiceTest {
+
+    public static String EVENTS_URL = "eventsUrl";
+
+    @Configuration
+    @Import(EventsCommunicationService.class) // A @Component injected with ExampleService
+    static class Config { }
+
+    @Autowired
+    EventsCommunicationService ecs;
+
+    @MockBean
+    RestTemplate restTemplateMock;
+
+    public EventsCommunicationServiceTest() {
+    }
+
+    @Before
+    public void setUp() {
+        ReflectionTestUtils.setField(this.ecs, "eventsUrl", EVENTS_URL);
+    }
+
+    @Test
+    public void makeCall_restTemplateIsProperlyCalled() {
```

Comment:
I know it would be better to have atomic tests like:
-> verify method type
-> verify content type is JSON
-> verify eventsUrl is prepended to path

but such verification would add even more code because we would use ArgumentCaptor. All in all, considering the nature of this method, I think this approach is cleaner and easier. Happy to discuss though